integration/networking/port_mapping_linux_test.go

```diff
@@ -400,3 +400,43 @@ func TestAccessPublishedPortFromRemoteHost(t *testing.T) {
 		}
 	}
 }
+
+func TestRestartUserlandProxyUnder2MSL(t *testing.T) {
+	skip.If(t, testEnv.IsRootless())
+
+	ctx := setupTest(t)
+
+	d := daemon.New(t)
+	d.StartWithBusybox(ctx, t)
+	defer d.Stop(t)
+
+	c := d.NewClientT(t)
+	defer c.Close()
+
+	const netName = "nat-time-wait"
+	network.CreateNoError(ctx, t, c, netName,
+		network.WithDriver("bridge"),
+		network.WithOption(bridge.BridgeName, netName))
+	defer network.RemoveNoError(ctx, t, c, netName)
+
+	ctrOpts := []func(*container.TestContainerConfig){
+		container.WithName(sanitizeCtrName(t.Name() + "-server")),
```

Comment:
Curious; was a name necessary for this test, or just for easier identification in logs?

Reply:
Easier identification - that was the only purpose here. But going forward, I'll also use it to have a single `defer c.ContainerRemove`.